documentation/docs/components/outputs/command-outputs/nym-node-migrate.md

```diff
@@ -0,0 +1,6 @@
+```sh
+** Attention **
```

Comment:
Here we go. . that's crunch message re migration.

Reply:
So I remove it?